server_admin/topics/user-federation/ldap.adoc

```diff
@@ -137,6 +137,14 @@ FullName Mapper::
   This allows you to specify that the full name of the user, which is saved in some LDAP attribute (usually `cn` ) will be mapped to `firstName` and `lastname` attributes in the {project_name} database.
   Having `cn` to contain full name of user is a common case for some LDAP deployments.
 
+NOTE: When registering new users in {project_name} and `Sync Registrations` switch is ON for the LDAP provider, the fullName mapper
+      allows the possibility of fallback to the username. This is especially useful in case of the Microsoft Active Directory. The common
+      setup for the MSAD is to configure `cn` LDAP attribute as fullName and at the same type, the `cn` is usually used as `RDN LDAP Attribute`
+      in the configuration of the LDAP provider. With this setup, the fallback to the username will be used. For example when you create
+      {project_name} user "john123" without firstName and lastName filled, then fullname mapper will save "john123" as the value of the `cn` in LDAP.
```

Comment:
change " without firstName and lastName filled" to "and leave firstName and lastName empty"

Reply:
I've changed this. Thanks